### ISSUE TITLE
Are there any plans to add support for Edge Chromium?

### ISSUE THREAD

**Issue by EOussama** [microsoft/vscode-cordova#588]
## Actual Behavior
1. The debugging mode always launches **Chrome**.

## Expected Behavior
1. Being able to customize the preferred browser, from which **Edge Chromium** can be chosen.

**Comment by SounD120** [comment]
Hi @EOussama and thanks for reaching us. When you are saying "customize the prefered browser" you mean add an ability to change default browser for simulate and serve debug scenarios?

**Comment by EOussama** [comment]
> Hi @EOussama and thanks for reaching us. When you say "customize the prefered browser", do you mean adding an ability to change the default browser for simulate and serve debug scenarios?

Yes, exactly that.

**Comment by SounD120** [comment]
Got it, thanks for your feedback, we'll consider that and manage in order of priorities.

**Comment by kenicenoel** [comment]
Agreed. I only have Google Chrome installed for this reason as the new Edge doesn't work with Cordova tools. I want to be able to use the extension with Edge and not Google Chrome.

**Comment by kenicenoel** [comment]
> Got it, thanks for your feedback, we'll consider that and manage in order of priorities.

I really hope this is prioritized as the new Edge is a Microsoft product and one would think their tooling should should be supported.

**Comment by Namiro** [comment]
+1 I totaly agree with @kenicenoel

**Comment by RedMickey** [comment]
Hi @EOussama, now Cordova Tools extension supports Edge Chromium browser for the `Serve to the browser (Ionic Serve)` debugging scenario. Simulate debugging scenarios (e.g. `"Simulate Android in browser`, `Run Browser`) haven't supported Edge Chromium yet, because they use the [`cordova-serve`](https://github.com/apache/cordova-serve) package, which doesn't support Edge Chromium, to run Cordova apps in the browser. We proposed [the PR](https://github.com/apache/cordova-serve/pull/40) to `cordova-serve` that adds support of Edge Chromium. So, when this PR is merged and a new version of the `cordova-serve` package is released, the extension should fully support Edge Chromium browser.
